docker/core/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM anapsix/alpine-java:8 AS build
+COPY codesearch-core.jar / 
+COPY Makefile /
+COPY scripts /scripts/
+VOLUME /data/rust/crates.io-index
+WORKDIR /data/rust/crates.io-index
+RUN git clone https://github.com/rust-lang/crates.io-index
+WORKDIR "/"
+RUN apk update && apk upgrade
+RUN apk add make \
+  git \
+  nodejs \
+  nodejs-npm \
+  go \
+  --no-cache musl-dev \
+  curl \
+  wget \
+  ruby \
+  ruby-json  
+RUN go get github.com/google/codesearch/cmd/...
+RUN rm -rf /var/cache/apk/*
+ENV PATH /root/go/bin:$PATH
+CMD ["make", "tables"]
```

Comment:
Why `tables` specifically?

Reply:
For my opinion i put this command because "make tables" the first command we should run, when starting the container, after that we can execute other commands.

Reply:
What If tables exists?

Reply:
@Som3 Than container will do nothing - because tables exists.